src/Storage/Controllers/SignController.cs

```diff
@@ -17,15 +17,15 @@ namespace Altinn.Platform.Storage.Controllers
     [ApiController]
     public class SignController : ControllerBase
     {
-        private readonly IInstanceService _instanceService;
+        private readonly ISigningService _signingService;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SignController"/> class
         /// </summary>
-        /// <param name="instanceService">A instance service with instance related business logic.</param>
-        public SignController(IInstanceService instanceService)
+        /// <param name="signingService">A instance service with instance related business logic.</param>
```

Comment:
While you're at it :p 
```suggestion
        /// <param name="signingService">An instance service with instance related business logic.</param>
```